pep-0692.rst

```diff
@@ -553,9 +457,6 @@ The `mypy type checker <https://github.com/python/mypy>`_ already
 `provides provisional support <pyrightProvisionalImplementation_>`__
 for `this feature <pyrightIssue3002_>`__.
 
-A proof-of-concept implementation of the CPython `grammar changes`_ described in
-this PEP is `available on GitHub <cpythonGrammarChangePoc_>`__.
-
 Rejected Ideas
 ==============
```

Comment:
The grammar change could be added to the rejected ideas.

Reply:
Yeah, that would be a good idea

Reply:
Agreed, added a new subsection.